### ISSUE TITLE
Emacs Sexy Icon V2 Checksum Mismatch

### ISSUE THREAD

**Issue by baron** [daviderestivo/homebrew-emacs-head#21]
```
==> Verifying eb68e5f126b77164faf3bb606c9f03198f3940c57e9791337368075ea97275cc--modern-icon-sexy-v2.icns checksum
Error: An exception occurred within a child process:
  ChecksumMismatchError: SHA256 mismatch
Expected: 5ff87050e204af05287b3baf987cc0dc558e2166bd755dd7a50de04668fa95af
  Actual: ecdc902435a8852d47e2c682810146e81f5ad72ee3d0c373c936eb4c1e0966e6
 Archive: /Users/me/Library/Caches/Homebrew/downloads/eb68e5f126b77164faf3bb606c9f03198f3940c57e9791337368075ea97275cc--modern-icon-sexy-v2.icns
To retry an incomplete download, remove the file above.
```

**Comment by daviderestivo** [comment]
Hi @baron,

thanks for reporting the issue. It has been fixed in 21d5d22e9a85ba9a03ab91d19e3761a40ffa13e4.

Regards,
Davide

**Comment by baron** [comment]
thanks so much for the fast work!
